AnimatedWordCloud/Utils/Config.py

```diff
@@ -85,3 +90,24 @@ def __init__(
         self.duration_per_frame = duration_per_frame
         self.n_frames_for_interpolation = n_frames_for_interpolation
         self.interpolation_method = interpolation_method
+        self.intermediate_frames_id = self._compute_intermediate_frames_id(
+            intermediate_frames_id
+        )
+
+    def _compute_intermediate_frames_id(
+        self, intermediate_frames_id: str | None
+    ) -> str:
+        """
+        returns intermediate_frames_id given from constructor
+
+        set random value if intermediate_frames_id is None
+
+        :param str|None intermediate_frames_id: intermediate_frames_id given
+        :return: intermediate_frames_id computed
+        :rtype: str
+        """
+
+        if intermediate_frames_id is None:
+            # set randomly
+            intermediate_frames_id = str(random.randint(0, 1000000000))
```

Comment:
You might make sure intermediate_frames_id is not really duplicated.
Under extreme circumstances, if over 1000000000 gifs were created, intermediate_frames_id woud definitely be duplicated.

Reply:
Not sure about this
The possibility is extremely low (0.0000001%) and it won't make error even it once used in the past (because `image_paths` are given by param, not getting dynamically)